src/vectorchord/advanced-features/rerank-in-table.md

```diff
@@ -0,0 +1,33 @@
+# Rerank In Table <badge type="tip" text="since v0.3.0" />
+
+To maximize performance, VectorChord stores vectors in both the index and the table.
```

Comment:
It's not for maximizing performance. It's how PostgreSQL indexes work. Fetching tuples from the heap is not what a correct index should do.